packages/deployment-service-ui/src/components/pages/api-key/list.tsx

```diff
@@ -0,0 +1,110 @@
+import { reapitConnectBrowserSession } from '@/core/connect-session'
+import {
+  configurationApiKeyApiCreateService,
+  configurationApiKeyApiDeleteService,
+  configurationApiKeyApiService,
+} from '@/platform-api/configuration-api'
+import { ReapitConnectSession, useReapitConnect } from '@reapit/connect-session'
+import { H3, Section } from '@reapit/elements'
+import { Button, Table } from '@reapit/elements'
+import { Loader } from '@reapit/elements/v3'
+import { ApiKeyInterface } from '@reapit/foundations-ts-definitions'
+import React, { useEffect, useState } from 'react'
+import { shleemy } from 'shleemy'
+
+export default () => {
+  const { connectSession } = useReapitConnect(reapitConnectBrowserSession)
+  const [apiKeys, setApiKeys] = useState<ApiKeyInterface[]>([])
+  const [loading, setLoading] = useState<boolean>(false)
+  const [creationLoading, setCreationLoading] = useState<boolean>(false)
+  const [deletionLoading, setDeletionLoading] = useState<string[]>([])
+
+  useEffect(() => {
+    const fetchApiKeys = async () => {
+      setLoading(true)
+      const serviceResponse = await configurationApiKeyApiService(connectSession as ReapitConnectSession)
+      setLoading(false)
+      if (serviceResponse) {
+        setApiKeys(serviceResponse)
+      }
+    }
+    if (connectSession) {
+      fetchApiKeys()
+    }
+  }, [connectSession])
+
+  const createApiKey = async () => {
```

Comment:
Usually I would say break these methods out so they can be tested but this is a quick and dirty app so no worries